### ISSUE TITLE
CR: Remove og:site_name Metatag

### ISSUE THREAD

**Issue by andyvanavery31** [NCIOCPL/cgov-digital-platform#4247]
### Background
Currently, the`og:site_name` metatag has `- National Cancer Institute` forced at the end of what is in Drupal for the site name. Therefore, NCP displays as `nationalcancerplan - National Cancer Institute` and Tech Transfer displays as `techtransfer - National Cancer Institute`. This is an issue for "unbranding". Based on research attached to this ticket and some below, this opengraph tag is not actually used anymore.  

### Request Description
Remove the `og:site_name` metatag from Drupal output, as well as remove any tokens associated with this specific use case.

### What is the Expected Change?
- The `og:site_name` is removed from markup.

### What is the Current Functionality?
- The `og:site_name` forces the `- National Cancer Institute` following the sitename tag

### Tasks

- [x] Investigate if `og:site_name` is still currently being used on social media platforms. Platforms to investigate are:
  - [x] Facebook
  - [x] X/Twitter
  - [x] Instagram
  - [x] YouTube
  - [x] LinkedIn
- [x] If `og:site_name` is not used on any of the above social media platforms, then it will be removed from all sources on the platform and the first section of Acceptance Criteria will be used

### Acceptance Criteria
```gherkin
Scenario: A user inspects any page on the digital platform
  Given the user is on a page on the digital platform
  When the user uses the browser inspect tool
    And uses the Find function in the inspect tool
    And searches for "og:site_name"
  Then there will be no results found
```

**Comment by bryanpizzillo** [comment]
### Investigation Notes
TL;DR; we might be better of removing this metatag altogether.
* The [Open Graph Protocol](https://ogp.me/) defines a set of HTML meta tags which can be used by Social Media platforms for sharing. 
   * These meta tags can control how the item you are sharing is presented on that platform in a users feed.
   * While Facebook created the standard, other platforms support these meta tags.
   * There are 4 required properties:
      * `og:title` - The title of your object as it should appear within the graph, e.g., "The Rock".
      * `og:type` - The [type](https://ogp.me/#types) of your object, e.g., "video.movie". Depending on the type you specify, other properties may also be required.
      * `og:image` - An image URL which should represent your object within the graph.
      * `og:url` - The canonical URL of your object that will be used as its permanent ID in the graph, e.g., "https://www.imdb.com/title/tt0117500/".
* `og:site_name` is actually an optional Open Graph field.
* Social Media Usage
   * [Facebook](https://developers.facebook.com/docs/sharing/webmasters#markup) - does not mention `og:site_name`

**Comment by blairlearn** [comment]
- [X, formerly known as Twitter](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup) does not mention `og:site_name`.
  - Testing tool: https://cards-dev.twitter.com/validator

**Comment by andyvanavery31** [comment]
YouTube does not support sharing content on their platform. Instagram does not support Open Graph.
